tools/bin/syndesis

```diff
@@ -683,7 +684,11 @@ run_minishift() {
         local memory=$(readopt --memory)
         local cpus=$(readopt --cpus)
         local disksize=$(readopt --disk-size)
-        minishift start --show-libmachine-logs=true --memory ${memory:-4912} --cpus ${cpus:-2} --disk-size ${disksize:-20GB}
+        local show_logs_arg=""
+        if [ $(hasflag --show-logs --show-log) ]; then
+            show_logs_args="--show-libmachine-logs=true "
+        fi
+        minishift start ${show_logs_arg:- }--memory ${memory:-4912} --cpus ${cpus:-2} --disk-size ${disksize:-20GB}
```

Comment:
@chirino hope you don't mind to make showing the logs optional, as you don't normally need this. Or which information is particular important to you ?